internal/observability/stream.go

```diff
@@ -37,33 +38,81 @@ var _ transport.StreamCloser = (*streamWrapper)(nil)
 
 type streamWrapper struct {
 	transport.StreamCloser
-	call call
+
+	call   call
```

Comment:
do we still need this field?

Reply:
yup, we need it for the `call.logStreamEvent` calls.